src/components/Page/components/Header/Header.tsx

```diff
@@ -349,10 +368,15 @@ function determineLayout({
       slots: {
         slot1: breadcrumbMarkup,
         slot2: pageTitleMarkup,
-        slot3: null,
+        slot3: (
+          <>
+            {actionMenuMarkup}
+            {primaryActionMarkup}
+          </>
+        ),
         slot4: paginationMarkup,
-        slot5: actionMenuMarkup,
-        slot6: primaryActionMarkup,
+        slot5: additionalMetaDataMarkup,
+        slot6: null,
```

Comment:
I think it should be safe to remove slot6 now but that can be done in a follow up